src/Table/Table.js

```diff
@@ -1,6 +1,7 @@
 import React, { useState, useEffect } from 'react';
 import PropTypes from 'prop-types';
 import uuidv1 from 'uuid/v1';
+import sortBy from 'sort-by';
```

Comment:
Could you use lodash.sortby intead? We are using lodash across blaze to do this kind of things.